pykechain/models/part.py

```diff
@@ -492,3 +492,27 @@ def as_dict(self):
             properties_dict[prop.name] = prop.value
         return properties_dict
 
+    def order_properties(self, list_property_names=None):
+        """
+        Order the properties of a part model using a list of property names.
+
+        Example
+        -------
+        >>> front_fork = client.scope('Bike Project').model('Front Fork')
+        >>> front_fork.order_properties(['Material', 'Height (mm)', 'Color'])
+
+        """
+        assert self.category == Category.MODEL
+
+        if isinstance(list_property_names, list):
+            order_dict = dict()
+
+            for prop in list_property_names:
+                order_dict[self.property(name=prop).id] = list_property_names.index(prop)
```

Comment:
in continuation of my comment above, the transformation from names to id's is already done here as the API requires the id's indeed. 

Please make it id and name agnostic